<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Panel3DVolume</class>
 <widget class="QFrame" name="Panel3DVolume">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>219</width>
    <height>607</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>3D体渲染成像</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/icons/Fatcow.qrc">
    <normaloff>:/fatcow/16x16/soil_layers.png</normaloff>:/fatcow/16x16/soil_layers.png</iconset>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QHBoxLayout" name="containerLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupRender">
       <property name="title">
        <string>渲染</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="1" colspan="2">
         <widget class="QComboBox" name="shaderQuality">
          <item>
           <property name="text">
            <string>低质量</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>智能切换</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>高质量</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="shaderQualityTitle">
          <property name="text">
           <string>渲染质量</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="alphaValue">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QCheckBox" name="measureFps">
          <property name="text">
           <string>测量FPS</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="suppressTitle">
          <property name="text">
           <string>抑制</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="fps">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="suppressValue">
          <property name="text">
           <string>100%</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSlider" name="suppress">
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
          <property name="pageStep">
           <number>10</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="alphaTitle">
          <property name="text">
           <string>不透明度</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSlider" name="alpha">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupSlices">
       <property name="title">
        <string>切片</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="2" column="1">
         <widget class="QSlider" name="sliceSliderZ">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSlider" name="sliceSliderY">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSlider" name="sliceSliderX">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="sliceEnableX">
          <property name="text">
           <string>X</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="sliceShowX">
          <property name="text">
           <string>显示</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="sliceShowY">
          <property name="text">
           <string>显示</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="sliceEnableZ">
          <property name="text">
           <string>Z</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="sliceShowZ">
          <property name="text">
           <string>显示</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="sliceEnableY">
          <property name="text">
           <string>Y</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="scaleEnable">
       <property name="title">
        <string>变形</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QSlider" name="scaleX">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSlider" name="scaleY">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSlider" name="scaleZ">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="scaleValueX">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="scaleValueY">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="scaleValueZ">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="scaleEnableX">
          <property name="text">
           <string>X缩放</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="scaleEnableY">
          <property name="text">
           <string>Y缩放</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="scaleEnableZ">
          <property name="text">
           <string>Z缩放</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupTrim">
       <property name="title">
        <string>表面修正</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="1">
         <widget class="QSlider" name="trimOffset">
          <property name="minimum">
           <number>-512</number>
          </property>
          <property name="maximum">
           <number>512</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="trimOffsetTitle">
          <property name="text">
           <string>同步深度</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="trimOffsetValue">
          <property name="text">
           <string>0%</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="3">
         <widget class="QDialogButtonBox" name="surfaceTrimButtons">
          <property name="standardButtons">
           <set>QDialogButtonBox::Apply</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="surfaceTrim">
          <property name="text">
           <string>启用修正</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ColorPaletteView" name="palette" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>35</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorPaletteView</class>
   <extends>QWidget</extends>
   <header>Imaging/ColorPaletteView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/icons/Fatcow.qrc"/>
  <include location="../../resources/icons/Fatcow.qrc"/>
 </resources>
 <connections/>
</ui>
